app/routers/ai_chat.py

```
@@ -0,0 +1,246 @@
+    user_tier: Literal["free", "premium"] = Field(
+        default="free", description="User subscription tier"
+    )
+    task_type: str = Field(default="text", description="Task type (text/embedding)")
+    force_provider: Optional[str] = Field(
+        None, description="Force specific provider (ollama/huggingface/openai)"
+    )
+
+    @validator("message")
+    def validate_message(cls, v):
+        if not v or not v.strip():
+            raise ValueError("Message cannot be empty")
```

Ruff (0.14.1)

Avoid specifying long messages outside the exception class

(TRY003)